### ISSUE TITLE
Broken devise's html template

### ISSUE THREAD

**Issue by denikus** [railsware/mailtrap-ruby#9]
Hey guys.

Just switched from mailgun to mailtrap and got problem with standard devise's emails.
Particularly with "password restoration" email template.
<img width="771" alt="image" src="https://user-images.githubusercontent.com/81196/209980133-34e58e1f-e711-4949-975f-37d4f99a23df.png">

I'm not 100% sure it's caused by mailtrap. But it seems like I didn't change anything else, except changing from one gem to another.

**Comment by i7an** [comment]
Hey!

Do you experience any issues with the emails you send via mailtrap?
There is [a very similar question](https://stackoverflow.com/a/4016098/876794) on stackoverflow.

But do let us know if you notice any functional defects (e.g. links do not work or content looks weird).

**Comment by i7an** [comment]
I think you're right about that. We really appreciate you sharing the results of your research. We'll discuss the problem in the team. Having said that, I can't promise any timeline.
Thanks again for the bug report. 🙇

**Comment by jaedung** [comment]
@i7an How is this issue going in your team?

I'm having the same problem.

**Comment by i7an** [comment]
@jaedung As a quick fix, please try switching to SMTP.
I will get back to you within a week with further updates.
Thank you for understanding.

**Comment by jaedung** [comment]
Hi @i7an 

Thanks for your reply.

I've switched to SMTP and it works smoothly but hopefully I can see it fixed for later usage. Thanks!

**Comment by i7an** [comment]
@jaedung I think we will be able to deliver the fix within two weeks. I'll post an update when its ready Thanks!

**Comment by i7an** [comment]
The fix is on the way just delayed. I will keep you posted.

**Comment by i7an** [comment]
Hi,

After experimenting with possible options, we decided to mitigate the issue with a series of small fixes to our email sending service and documentation:
* You will no longer see two `Content-Transfer-Encoding` headers in emails;
* [API documentation](https://railsware.stoplight.io/docs/mailtrap-api-docs/67f1d70aeb62c-send-email) now explicitly states that `Content-Transfer-Encoding` header is ignored;
* A new chapter is added to the [README](https://github.com/railsware/mailtrap-ruby#content-transfer-encoding) explaining how to deal with `Content-Transfer-Encoding` headers.

In short, please do not expect Mailtrap API to mirror all the features of the SMTP protocol. If you need your emails to be sent with a specific `Content-Transfer-Encoding`, consider switching to Mailtrap SMTP.

Thank you for your patience and sorry if your expectation were different 🙏

If you have a different opinion on this matter, please share it as we might consider implementing your suggestions.
